README.md

```diff
@@ -17,13 +17,18 @@ virtualenv --python=python2 .venv
 pip install tox
 ```
 
+You also need to install ansible. For that purpose execute the following command:
+
+```
+sudo apt install ansible
```

Comment:
Maybe we need to omit this instruction. As was @ylobankov noticed it won't work for Mac.